recipes/conifer/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "Conifer" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/Ivarz/Conifer/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 741ed0e04fe46364d154b09b32d6c76dbd210ba84fc2eb357070d16d2eedf2f1
+  
+  - url: https://github.com/ThrowTheSwitch/Unity/archive/5e9acef74faffba86375258d822f49d0b1173b5e.zip
+    sha256: 33d277423179d15f439a1bda21df94bafa43c9179cc496ccf799495abb3eb5ad
+    folder: third_party/Unity
+  
+  - url: https://github.com/troydhanson/uthash/archive/8e67ced1d1c5bd8141c542a22630e6de78aa6b90.zip
+    sha256: b6405dbb2ef469a3147892906214a369a0b134560ed943a20e450c21c7b6720d
+    folder: third_party/uthash
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - zlib
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+test:
+  commands:
+    - conifer -v
```

Comment:
shouldn't the other binaries also be tested? e.g. show_ancestors

Reply:
Added them here, originally avoided due to exiting with non-zero on usage. 

Thanks again @bgruening